src/components/App.js

```diff
@@ -0,0 +1,37 @@
+import ButtonPannel from './ButtonPanel';
+import Display from './Display';
+import Button from './Button';
+
+function App() {
+  return (
+    <div className="App">
+      <div className="App__calculator">
+        <Display />
+        <ButtonPannel>
+          <Button name="AC" />
+          <Button name="X²" />
+          <Button name="%" />
+          <Button name="/" />
+          <Button name="9" />
+          <Button name="8" />
+          <Button name="7" />
+          <Button name="*" />
+          <Button name="6" />
+          <Button name="5" />
+          <Button name="4" />
+          <Button name="+" />
+          <Button name="3" />
+          <Button name="2" />
+          <Button name="1" />
+          <Button name="-" />
+          <Button name="0" />
+          <Button name="." />
+          <Button name="+/-" />
+          <Button name="=" />
+        </ButtonPannel>
```

Comment:
- [ ] Rather than this. Kindly group the buttons into sections and then map through it. It would be neater
```
Group 1: AC, +/-, %, ÷
Group 2: 7, 8, 9, X
Group 3: 4, 5, 6, -
Group 4: 1, 2, 3, +
Group 5: 0, ., =
```

Reply:
Fixed